### PR TITLE
AFKR-7: Added french translations for the appointment scheduling and bed management apps

### PR DESCRIPTION
**Pull request by rubailly**
- Keys for the home app icons are provided with [this commit](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/6/commits/40ed9cb3e6276b719146bb95b0c35e1f125e1e33)

- Both [default-config](https://github.com/Bahmni/default-config/tree/master/openmrs/i18n/ipd) an [bahmniapps](https://github.com/Bahmni/openmrs-module-bahmniapps/tree/master/ui/app/i18n/ipd) don't provide a `locale_fr.json` file.  So when I created it for `bahmni-config-easydoc` I combined the content of the two files in one file. See [this commit](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/6/commits/961654d44d1622d0dbeb22883daa11f29148b1d2).

- There is a french translation file for the appointment scheduling app that is provided by [bahmniapps](https://github.com/Bahmni/openmrs-module-bahmniapps/blob/master/ui/app/i18n/appointments/locale_fr.json). With [this commit ](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/6/commits/961654d44d1622d0dbeb22883daa11f29148b1d2), I'm trying to rather provide the file via bahmni_config.

### CODE REVIEW

File: configuration/openmrs/i18n/appointments/locale_fr.json [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  "CHECK_IN_APPOINTMENT": "Check-in",
+  "COMPLETE_APPOINTMENT": "Complete",
+  "MISSED_APPOINTMENT": "Missed",
+  "EDIT_APPOINTMENT": "Edit",
+  "CANCEL_APPOINTMENT": "Cancel",
+  "CURRENT_DATE": "Aujourd'hui",
+  "DAY": "Jour",
+  "WEEK": "Semaine",
+  "TOTAL_KEY": "Total",
+  "MISSED_KEY": "missed",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/appointments/locale_fr.json]: Could you replace by "Manqué" or "manqué" .

File: configuration/openmrs/i18n/appointments/locale_fr.json [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  "PLACEHOLDER_SELECT_PATIENT": "Select a patient",
+  "APPOINTMENT_SLOT": "Slot",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/appointments/locale_fr.json]: Replace by "Créneau"

File: configuration/openmrs/i18n/appointments/locale_fr.json [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  "EDIT_APPOINTMENT_KEY": "Edit Appointment",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/appointments/locale_fr.json]: "Modifier le rendez-vous"

File: configuration/openmrs/i18n/appointments/locale_fr.json (hunk 1 above)
Comment (rubailly) [reply configuration/openmrs/i18n/appointments/locale_fr.json]: [Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/6/commits/427449b1d84cce0b217ef35f341a63c96db84f90)

File: configuration/openmrs/i18n/appointments/locale_fr.json (hunk 2 above)
Comment (rubailly) [reply configuration/openmrs/i18n/appointments/locale_fr.json]: [Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/6/commits/427449b1d84cce0b217ef35f341a63c96db84f90)

File: configuration/openmrs/i18n/appointments/locale_fr.json (hunk 3 above)
Comment (rubailly) [reply configuration/openmrs/i18n/appointments/locale_fr.json]: [Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/6/commits/427449b1d84cce0b217ef35f341a63c96db84f90)

**Pull request merged**